maxrange: 3.0
i2c_channel_base: 0x3
